packages/data-stores/src/reader/helpers/index.ts

```diff
@@ -8,6 +8,16 @@ type callApiParams = {
 	isLoggedIn?: boolean;
 };
 
+// Get cookie named subkey
+const getSubkey = () => {
+	const subkey = document.cookie
```

Comment:
This could be changed to use the cookie library, something like:
```
const cookies = cookie.parse( document.cookie );
return cookies.subkey;
```

Reply:
Thanks for the suggestion, Paulo! I will implement the change.

Reply:
Addressed in https://github.com/Automattic/wp-calypso/pull/74830/commits/cd016de255b8afd160a4a1c68e9c792347beea59.